docs/core/docker/docker-basics-dotnet-core.md

```diff
@@ -0,0 +1,280 @@
+---
+title: Learn Docker Basics with .NET Core
+description: A Docker and .NET Core Basic Tutorial
+keywords: .NET, .NET Core, Docker, Tutorial
+author: jralexander
+ms.author: johalex
+ms.date: 10/11/2017
+ms.topic: article
+ms.prod: .net-core
+ms.technology: dotnet-docker
+ms.devlang: dotnet
+ms.assetid: 03c28597-7e73-46d6-a9c3-f9cb55642739
+---
+
+# Learn Docker Basics with .NET Core
+
+This tutorial teaches the Docker container build and deploy tasks for a .NET Core application. During the course of this tutorial, you learn:
+
+> [!div class="checklist"]
+> * How to create a Dockerfile
+> * How to create a .NET Core app.
+> * How to deploy your app into a Docker container.
+
+The [Docker platform](https://docs.docker.com/engine/docker-overview/#the-docker-platform) uses the [Docker Engine](https://docs.docker.com/engine/docker-overview/#docker-engine) to quickly build and package apps as [Docker images](https://docs.docker.com/glossary/?term=image). These images are written in the [Dockerfile](https://docs.docker.com/glossary/?term=Dockerfile) format to be deployed and run in a [layered container](https://docs.docker.com/engine/userguide/storagedriver/imagesandcontainers/#container-and-layers).
+
+## .NET Core: Easiest way to get started
+
+Before creating the Docker image, you need an application to containerize. You can create it on Linux, MacOS, or Windows. The quickest and easiest way to do that is to use .NET Core.
+
+If you're unfamiliar with the .NET Core CLI toolset, read the [.NET Core SDK overview](../tools/index.md).
+
+You can build both Windows and Linux containers with [multi-arch based tags](https://github.com/dotnet/announcements/issues/14).
+
+## Your first .NET Core Docker app
+
+### Prerequisites
+
+#### .NET Core 2.0 SDK
+
+* Install [.NET Core SDK 2.0](https://www.microsoft.com/net/core).
+
+* Install your favorite code editor, if you haven't already.
```

Comment:
I'd recommend Visual Studio and VS Code here, if a reader doesn't have a favorite editor.